srcpkgs/The-Powder-Toy/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'The-Powder-Toy'
 pkgname=The-Powder-Toy
-version=96.2.350
+version=98.2.365
 revision=1
 build_style=meson
-configure_args="-Dx86_sse=sse2 -Dworkaround_gcc_no_pie=false
+configure_args="-Dx86_sse=sse2
  -Dlua=$(vopt_if luajit luajit lua5.1)"
 hostmakedepends="pkg-config"
-makedepends="SDL2-devel fftw-devel zlib-devel libcurl-devel
+makedepends="SDL2-devel fftw-devel zlib-devel libcurl-devel bzip2-devel
+ libpng-devel jsoncpp-devel
```

Comment:
Add those new deps in alphabetical order

Reply:
none of the other ones are in alphabetical order. Am I missing something?

Reply:
True that, leave it as-is and I'll try to remember for next update